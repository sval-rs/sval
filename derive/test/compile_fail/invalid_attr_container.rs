#[macro_use]
extern crate sval_derive;

#[derive(Value)]
#[sval(not_an_attr = "true")]
pub struct Record {
    a: i32,
}

fn main() {}
