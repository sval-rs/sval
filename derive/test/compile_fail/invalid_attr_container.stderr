error: proc-macro derive panicked
 --> compile_fail/invalid_attr_container.rs:4:10
  |
4 | #[derive(Value)]
  |          ^^^^^
  |
  = help: message: unsupported attribute `not_an_attr` on container
